cmd/trigger_ui/main.go

```diff
@@ -74,6 +75,12 @@ func singleValueArrayFromRows(rows *sql.Rows, err error) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer func() {
+		err := rows.Close()
+		if err != nil {
+			log.Default().Println(err)
+		}
+	}()
```

Comment:
FYI: I think it's quite common to ignore the error here (i.e. `defer rows.Close()`) IIRC because they are rare and usually just indicate that you after all did not need to close it.